scriptmodules/emulators/dosbox-staging.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="dosbox-staging"
+rp_module_desc="DOS emulator (enhanced and modern DOSBox fork)"
```

Comment:
Wouldn't it be better to use the website descriotion (_DOS/x86 emulator focusing on ease of use_) ? `fork` is also fine, just a suggestion.

Reply:
I propose the following compromise: modern DOS/x86 emulator focusing on ease of use (modern is important given the massive work done to modernize the codebase).

Reply:
Looks fine.